src/modules/universal-discovery/components/content-meta-preview/content.meta.preview.component.js

```diff
@@ -166,7 +166,11 @@ export default class ContentMetaPreviewComponent extends Component {
      * @memberof ContentMetaPreviewComponent
      */
     removeBookmark() {
-        console.log('CMP - bookmark removed');
+        const { onBookmarkRemoved, data } = this.props;
+
+        // TODO: add query to REST, after REST ready
```

Comment:
`console.warn('TODO: <your-message>')` - this way you'll never forget about adding it ;)